index.js

```diff
@@ -147,26 +190,32 @@ const line = []
 
 // ***** Scope & Closures - Question 1 *****
 
+ticketNumber = 0
 
-// *** Uncomment the lines below to test
-// console.log("%cScope & Closures - Question 1", "color: red")
+function takeATicketNumber(line) {
+  ticketNumber ++
+  line.push(ticketNumber)
+  return `Welcome. You are ticket number ${ticketNumber}`
+}
```

Comment:
Be careful on line 193 here, if you declare a variable without `let` or `const` it will become a global variable (even if you declare it inside a function). To make sure that the ticket variable here is scoped to the function, use `let`. Otherwise, looks good!